flink-end-to-end-tests/flink-confluent-schema-registry/src/main/java/org/apache/flink/schema/registry/test/TestAvroConsumerConfluent.java

```diff
@@ -49,14 +49,12 @@ public static void main(String[] args) throws Exception {
 			System.out.println("Missing parameters!\n" +
 				"Usage: Kafka --input-topic <topic> --output-string-topic <topic> --output-avro-topic <topic> " +
 				"--bootstrap.servers <kafka brokers> " +
-				"--zookeeper.connect <zk quorum> " +
```

Comment:
Hi @becketqin , could you help to review this? 
This PR also remove `zookeeper.connect` properties for non-table-modules.